addons/attributes/stringtable.xml

```diff
@@ -238,5 +238,11 @@
             <English>Abilities</English>
             <Polish>Zdolności</Polish>
         </Key>
+        <Key ID="STR_ZEN_Attributes_OpenAttributesDisplay">
+            <English>Open Attributes Display</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_OpenAttributesDisplay_Description">
+            <English>Opens the Zeus attributes display for the mouse-overed or selected entity.</English>
```

Comment:
Should add how the hovering behaviour works. Maybe something like: "... display for the selected entities.\nHovering over a selected entity can be used to specify the exact entity to open the display for."

Reply:
How's this?
```
    <Key ID="STR_ZEN_Attributes_OpenAttributesDisplay_Description">
            <English>Opens the Zeus attributes display for the selected entity.\nWhen multiple entitie are selected (i.e. due to selecting a Group), the moused-over entity will be used.</English>
        </Key>
```

Reply:
Looks good. Should lowercase "group" and maybe use "hovered" instead of "mouse-over".

Reply:
https://github.com/zen-mod/ZEN/pull/437/commits/1a64b2ab08138927e00752034561b559e635ee1d